packages/node_modules/@ciscospark/widget-space/src/actions.js

```diff
@@ -1,4 +1,22 @@
+import {isUuid} from '@ciscospark/react-component-utils';
+
+export const FETCH_SPACE_ERROR = `widget-space/FETCH_SPACE_ERROR`;
```

Comment:
Would it make sense to put all of this into a redux-module, that way the logic doesn't have to be repeated each top level widget?